src/fisher.jl

```diff
@@ -36,11 +36,37 @@ immutable FisherExactTest <: HypothesisTest
 
     # conditional maximum likehood estimate of odd ratio
     ω::Float64
+end
 
-    function FisherExactTest(a::Int, b::Int, c::Int, d::Int)
```

Comment:
Shouldn't move this function out of the type definition: this would make it an outer constructor rather than an inner constructor. Just attach the docstring to the type itself.